### PR TITLE
Fixes for running 64-bit Hardware

### PR DESCRIPTION
**Pull request by aroelke**
Use `riscv64-unknown-elf-gdb` to execute code on a 64-bit AP and fix the AP UART message that is expected by the runtime to know if the test is complete.

### CODE REVIEW

File: runtime/modules/isp_vcu118.py [hunk 1]
```diff
@@ -279,7 +282,7 @@ def runStock(exe_path, ap, openocd_log_file, gdb_log_file,
         return isp_utils.retVals.FAILURE
 
     logger.debug("Spawning gdb")
-    gdb = threading.Thread(target=gdb_thread, args=(exe_path, gdb_log_file))
+    gdb = threading.Thread(target=gdb_thread, args=(exe_path, gdb_log_file, "rv32"))
```
Comment (JohnPMerrill) [inline runtime/modules/isp_vcu118.py:285]: Don't hard code "rv32". Use the `arch` argument instead

File: runtime/modules/isp_vcu118.py [hunk 2]
```diff
@@ -259,6 +261,7 @@ def runPipe(exe_path, ap, pex_tty, pex_log, openocd_log_file,
         while True:
             pass
 
+    logger.info("waiting for pex and ap to finish")
```
Comment (JohnPMerrill) [inline runtime/modules/isp_vcu118.py:264]: logger.debug

File: runtime/modules/isp_vcu118.py [hunk 3]
```diff
@@ -235,6 +236,7 @@ def runPipe(exe_path, ap, pex_tty, pex_log, openocd_log_file,
     
     logger.info("Sending flash init file {} to {}".format(flash_init_image_path, pex_tty))
     pex_serial.write(open(flash_init_image_path, "rb").read())
+    logger.info("Done writing init file")
```
Comment (JohnPMerrill) [inline runtime/modules/isp_vcu118.py:239]: logger.debug

File: runtime/templates/vcu118/isp_utils.c [hunk 4]
```diff
@@ -32,7 +32,13 @@ void isp_test_device_fail(void)
 uint64_t isp_get_cycle_count(uint32_t *result_hi, uint32_t *result_lo)
 {
 #if __riscv_xlen == 64
-	return read_csr(mcycle);
+	uint64_t cycle;
```
Comment (natestuder) [inline runtime/templates/vcu118/isp_utils.c:35]: What's the reason for this change?  Doesn't read_csr already do this?

Comment (aroelke) [reply runtime/templates/vcu118/isp_utils.c:35]: `mcycle` is undefined in this file and the headers it includes.

File: runtime/modules/isp_vcu118.py (hunk 1 above)
Comment (arunthomas) [reply runtime/modules/isp_vcu118.py]: Fixed

File: runtime/modules/isp_vcu118.py (hunk 3 above)
Comment (arunthomas) [reply runtime/modules/isp_vcu118.py]: Fixed
